src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Cluster Virtualization Overview
```

Comment:
I see two sections from the menu in the preview: "Setting Scopes" and "Metric Scopes". I would consolidate those two into one.

Reply:
Well, settings and metrics are different sorts of things. That's why they are separate. Eventually the scopes for metrics and settings will be added to the general metrics and settings documentation, and these pages will no longer be needed. They are segregated right now because they are only interesting to users with CV enabled.